src/api/routers/participantsRouter.ts

```diff
@@ -240,16 +241,18 @@ export function createParticipantsRouter() {
 
       const sharingParticipants = await addSharingParticipants(
         participant.siteId,
-        newParticipantSites
+        newParticipantSites,
+        newTypes
       );
 
       await updateAuditTrailToProceed(auditTrail.id);
-      return res.status(200).json(sharingParticipants.map(mapParticipantToAvailableParticipant));
```

Comment:
could we now remove `mapParticipantToAvailableParticipant`?

Reply:
done!